app/models/manageiq/providers/openstack/cloud_manager/template.rb

```diff
@@ -72,4 +72,21 @@ def has_proxy?
   def requires_storage_for_scan?
     false
   end
+
+  def raw_delete_image
+    ext_management_system.with_provider_connection(:service => 'Compute') do |service|
```

Comment:
@aufi  btw do we need to change `:service => 'Compute' ` to `:service => 'Image'` here as for [create](https://github.com/ManageIQ/manageiq-providers-openstack/pull/89/files#diff-dc700e83d755312bb00d3e67ab440984R77) action?

Reply:
@andyvesel Yes please, create a new PR. I missed it before.

Reply:
Ok
https://github.com/ManageIQ/manageiq-providers-openstack/pull/103